### ISSUE TITLE
python31-branch ready to be tested :)

### ISSUE THREAD

**Issue by Morgawr** [pikpik/minix-pkgsrc#5]
It's the first time I use git and I'm not very good at it, however I was able to build and install python31, it works like python24 (aka no shared libs, but at least it's there).

Let me know if you have problems, I wanted to add a file with git with explanation of the changes but git isn't working right for me at the moment (I tried "git add myfile" and then "commit" but it said there were no changes...).

Enjoy.

**Comment by pikpik** [comment]
Hi,

Awesome! It builds (`bmake install`) and runs (`python3.1`) for me without errors.

There are quite a few warnings about (something like) `-arch=i586` not being supported by `ld`. I suppose we should ask about this on IRC at #minix and maybe #pkgsrc.

I added comments to the last commit. It would be good to separate changes to different packages into different branches. Especially if they aren't related to each other.

I guess we'll have some things to revert when shared libraries are supported. But until then, this is wonderful. :)

Thanks a lot!

**Comment by Morgawr** [comment]
Yeah, I know.. those warnings are also in python24 so that's not just a 3.1 issue...

**Comment by pikpik** [comment]
Ah, ok. That's good then.
